AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginCPKTests/PrimaryKey/README.md

```diff
@@ -1,12 +1,21 @@
 # DataStore Primary Key Integration Tests
```

Comment:
I moved the CPK tests completely out of the Integration test target, into its own target. This is more aligned with what's in `main`, 1 test target per backend, and avoids conflicting generated model files (TransformerV2 and this test also uses Post4V2, but should not be shared since the feature flags are different. TransformerV2 does not use CPK feature flag, which is what this test is exercising with Post4V2/Comment4V2 models.)